src/containers/NonBitGoRecoveryCoin/NonBitGoRecoveryCoin.tsx

```diff
@@ -1024,6 +1024,63 @@ function Form() {
           }}
         />
       );
+    case 'osmo':
+    case 'tosmo':
+      return (
```

Comment:
if there is no difference between this and atom, then just add these coins also in above case only.